src/tests/ftest/io/ior_small.py

```diff
@@ -64,6 +64,12 @@ def test_ior_small(self):
         for oclass in obj_class:
             self.ior_cmd.dfs_oclass.update(oclass)
             for api in apis:
+                # Skip test issue seen while advancing hdf5 vol-daos code
+                # That is still needed due to support for new libdaos v1 API
+                if (api == "HDF5-VOL" and
+                        transfer_block_size[0] == "1M" and
+                        transfer_block_size[1] == "32M"):
+                    self.cancelForTicket("DAOS-6427")
```

Comment:
Just noting the [discussion that this is not working](https://github.com/daos-stack/daos/pull/3935#issuecomment-756178899) and the [failed test result](https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-3935/38/testReport/junit/(root)/IorSmall/Test___Functional_Hardware_Large___1___io_ior_small_py_IorSmall_test_ior_small_transport_config_container_dfuse_transport_config_hdf5_vol_hosts_np_16_iorflags_pool_servers_transport_config_c71f/).